builder/azure/arm/builder.go

```diff
@@ -254,8 +254,14 @@ func (b *Builder) Run(ui packer.Ui, hook packer.Hook, cache packer.Cache) (packe
 		return nil, errors.New("Build was halted.")
 	}
 
+	osType := "Linux"
+	if b.config.OSType == constants.Target_Windows {
+		osType = "Windows"
```

Comment:
Any reason not to use osType = constants.Target_Linux or constants.Target_Windows?